### ISSUE TITLE
Add sidebar

### ISSUE THREAD

**Issue by pulkitvyas08** [labs-vision/Karyakram-Frontend#7]

**Comment by rakshit087** [comment]
There is an Sidebar already, shall I replace it?

**Comment by pulkitvyas08** [comment]
Yes. The current sidebar is basically a dev navigation bar made in #44. Refactor it to something like `@components/DevNav` and add the proper sidebar UI in place of the current one
